src/js/components/Link/StyledLink.js

```diff
@@ -24,30 +24,30 @@ const sizeStyle = props => {
   `;
 };
 
-const StyledAnchor = styled.a`
+const StyledLink = styled.a`
   box-sizing: border-box;
   ${props => sizeStyle(props)}
   color: ${props =>
-    normalizeColor(props.colorProp || props.theme.anchor.color, props.theme)};
+    normalizeColor(props.colorProp || props.theme.link.color, props.theme)};
```

Comment:
can we also make these changes backwards compatible? easiest way to do is first check in theme.link. if its not available we search in theme.anchor.
and lets add a comment in this file to remove this compatibility check in next major